modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,98 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify that it's possible to send an e-mail notification with Action (Standalone)"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an Email Notification Template and Custom Object with an Action (Standalone),") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
```

Comment:
This should be an email address `test@liferay.com` since that's where we're sending it from